.github/workflows/continuous-integration.yml

```diff
@@ -246,5 +253,5 @@ jobs:
                                --command "${{ env.COMMAND-E2E-MGPU }}" \
                                --remote https://github.com/'${{ github.event.pull_request.head.repo.full_name }}' \
                                --source-ref '${{ github.event.pull_request.head.sha }}' \
-                               --wait    
-
+                               --wait
+        
```

Comment:
Same here, add newline.

Reply:
The 257 is the new line, right?

Reply:
Yup just add a newline at the end of the file.

Reply:
Unresolving, the trailing whitespace on line 263 is still there.